inventory/group_vars/all.yml

```diff
@@ -16,7 +16,7 @@ deployment_user: ofn-deploy
 ssh_pubkey: "{{ lookup('file', '~/.ssh/{{ ssh_pubkey_name }}') }}"
 
 # You can paste the repo/branch straight from a github PR here, like: "Contributor22:new_feature"
-github_key: openfoodfoundation:master
+github_key: openfoodbrazil:master
```

Comment:
We can't merge this change to `master`...

Reply:
I see you have just one commit in the fork at https://github.com/openfoodbrazil/openfoodnetwork and it's just for adding some images. 

You can deploy the app from the main repo and customise those images afterwards, there's no need for a fork. Once the server is deployed you can upload images (like the homepage banner) and customise some of the content quite easily from the UI. :+1: